inputFiles/compositionalMultiphaseWell/simpleCo2InjTutorial_base.xml

```diff
@@ -85,11 +85,12 @@
 
   <!-- SPHINX_FIELD_CASE_Co2_CONSTITUTIVE -->
   <Constitutive>
-    <CO2BrineFluid
+    <CO2BrinePhillipsFluid
```

Comment:
```suggestion
    <CO2BrineEzrokhiFluid
```
if I understood well based on the `pvtliquid.txt` in `compositionalMultiphaseWell` (but we can check all this when we do the rebaseline anyway)

Reply:
Thanks for noticing this @francoishamon! I intended to keep the previous behaviour, so `pvtliquid.txt` should have been with PhillipsBrineDensity. My bad, I`m going to restore the previous file.